### ISSUE TITLE
[Wikimedia] TypeError: d.query is undefined

### ISSUE THREAD

**Issue by nicolas-raoul** [tbo47/tbo47.github.io#2]
Thanks for making https://tbo47.github.io/wikimedia/

Unfortunately I get this error:
```
Uncaught (in promise) TypeError: d.query is undefined
    wikimediaQuery https://tbo47.github.io/ez-opendata.js:134
    renderMap https://tbo47.github.io/wikimedia/index.js:7
    async* https://tbo47.github.io/wikimedia/index.js:13
    async* https://tbo47.github.io/wikimedia/index.js:18
[ez-opendata.js:134:5](https://tbo47.github.io/ez-opendata.js)
    <anonymous> https://tbo47.github.io/wikimedia/index.js:18
    AsyncFunctionThrow self-hosted:857
    (Async: async)
    <anonymous> https://tbo47.github.io/wikimedia/index.js:18
```

Latest Firefox on Linux, centered on Tokyo.

![Screenshot from 2023-10-05 14-14-57](https://github.com/tbo47/tbo47.github.io/assets/99590/dc3d050a-f81c-47b4-9d7c-af6cb3656832)

**Comment by tbo47** [comment]
Can you tell me what the `d` variable looks like for you?
![image](https://github.com/tbo47/tbo47.github.io/assets/1559534/a90d8335-f72b-4943-a992-422c155cb5a8)

**Comment by tbo47** [comment]
oh, I see what the problem is. I will fix it shortly.

**Comment by nicolas-raoul** [comment]
It is working now, thanks a lot! :-)

**Comment by tbo47** [comment]
I just released 1.0.1 with this bug fixed https://github.com/tbo47/ez-opendata
